addons/web_editor/static/src/js/wysiwyg/widgets/color_palette.js

```diff
@@ -193,8 +201,14 @@ const ColorPaletteWidget = Widget.extend({
      * @param {string} color Color to add to the cuustom colors
      */
     _addCustomColor: function (existingColors, color) {
-        const normColor = ColorpickerDialog.normalizeCSSColor(color);
-        if (color && !existingColors.has(normColor)) {
+        if (!color) {
+            return;
+        }
+        let normColor = ColorpickerDialog.normalizeCSSColor(color);
+        if (!ColorpickerDialog.isCSSColor(normColor)) {
```

Comment:
Should not you check if it is a css color before normalizing it ?

Reply:
`normalizeCSSColor` and `isCSSColor` both use `convertCSSColorToRgba` to know if the color is a css Color. Still I can call `normalizeCSSColor` only once instead of twice like I do here.

Reply:
Yep, that's exactly the point... they do the same stuff so do it only once x)